CMakeLists.txt

```diff
@@ -58,20 +58,17 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 # Disable compiler specific extensions like gnu++11.
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-set(OPENROAD_VERSION ${PROJECT_VERSION})
-message(STATUS "OpenROAD version: ${OPENROAD_VERSION}")
-
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
-# Get version string in OPENROAD_GIT_DESCRIBE
+# Get version string in OPENROAD_VERSION
```

Comment:
OPENROAD_VERSION is used in the main.cc to set the commandline string for tcl when using tclreadline. This would result in that line being rather long. It might be better to either remove the dependency in main.cc or provide a short VERSION name for that.

Reply:
good point - I would drop it and just use "openroad > "

Reply:
@vvbandeira can you make that change before merging?

Reply:
Yes, I will make the change.

Reply:
possible fix on e3681d8